.circleci/config.yml

```diff
@@ -139,15 +139,15 @@ jobs:
           path: ~/ibc
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum "package-lock.json" }}
```

Comment:
I guess v1 just carried over from DAO DAO? would remove v1 from the whole file if its not needed here, or else might cause confusion (of where is v2?)

Reply:
> I guess v1 just carried over from DAO DAO? would remove v1 from the whole file if its not needed here, or else might cause confusion (of where is v2?)

This is actually not from DAO DAO! We use GitHub CI which I actually prefer.